bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/internal/DeviceManagerFactory.java

```diff
@@ -103,9 +111,15 @@ public void initialize() {
             }, scheduler);
         }).whenComplete((devManagerWrapper, th) -> {
             if (th != null) {
-                logger.warn("Failed to initialize DeviceManager: {}", th.getMessage());
+                if (th.getCause() instanceof DBusException) {
+                    // we might be on a system without DBus, such as macOS or Windows
+                    logger.debug("Failed to initialize DeviceManager: {}", th.getMessage());
+                } else {
+                    logger.warn("Failed to initialize DeviceManager: {}", th.getMessage());
+                }
```

Comment:
This should be the only change required for the fix.

Reply:
After closer inspection it looks like a try/catch for the CompletionException on line 65 would be required as well.